build/app/cli.js

```diff
@@ -65,8 +65,8 @@ program
 		const script = './node_modules/@financial-times/n-ui/scripts/build-sass.sh';
 		const commands = {
 			jsOnly: `'webpack --bail --config ${webpackConfPath} ${options.production ? '-p' : ''}'`,
-			sassOnly: `export CSS_SOURCE_MAPS=${!options.production} && ${cssEntryPoints
-				.map(([target, entry]) => `'${script} ${entry} ${target}'`)
+			sassOnly: `${cssEntryPoints
+				.map(([target, entry]) => `'export CSS_SOURCE_MAPS=${!options.production} && ${script} ${entry} ${target}'`)
```

Comment:
By the way, this will `export CSS_SOURCE_MAPS` multiple times, right? One for each iteration in the `map()` ..?

Reply:
Yup, _but_, if concurrently works the way I assume it would work (by spawning a new process for each command), I think it would have to be exported for each one anyway. (But that's a wild guess)